frontend/.env.example

```diff
@@ -1,3 +1,3 @@
 # API
 #
-REACT_APP_API_ORIGIN_URL=/api/v1
+REACT_APP_API_ORIGIN_URL=http://localhost:5001/api/v1
```

Comment:
Shouldn't use http://localhost:5001 in this place. Please give some info on why you needed it.

Reply:
because otherwise requests start with /api/v1 and go the frontend site instead of backend (which is on another url)

Reply:
Why did it work before?